db/migrate/20210510130434_create_participant_events.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateParticipantEvents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :participant_events, id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
Are we sure we want a random uid to be the default here? :/

Reply:
Should we lean on postgreSQL to give us the UUIDs https://pawelurbanek.com/uuid-order-rails

Reply:
> Are we sure we want a random uid to be the default here? :/

Standard for paper_trail (as also used in the "versions" table that it was based on.

Reply:
You should be able to omit all of this. We have a gem that sorts this for us: https://github.com/DFE-Digital/early-careers-framework/blob/develop/Gemfile#L30

cf the other migrations